tests/fixtures/drainpipe-task-upgrade/Taskfile.yml

```diff
@@ -10,6 +10,7 @@ includes:
   snapshot: ./vendor/lullabot/drainpipe/tasks/snapshot.yml
   test: ./vendor/lullabot/drainpipe/tasks/test.yml
   javascript: ./vendor/lullabot/drainpipe/tasks/javascript.yml
+  tugboat: ./vendor/lullabot/drainpipe/tasks/tugboat.yml
```

Comment:
Remove this line. This is checking that we're not breaking pre-existing installs, not this new feature.